Extensions/show_more.js

```diff
@@ -424,19 +424,29 @@ XKit.extensions.show_more = new Object({
 
 		if (typeof $(m_obj).attr('href') === "undefined") { return; }
 
-		var m_url = $(m_obj).attr('href').toLowerCase();
+		var m_url = $(m_obj).attr('href');
 
-		if (m_url.substring(m_url.length - 1) === "/") {m_url = m_url.substring(0, m_url.length - 1); }
+		/*Test for a link to a tumblr blog in a form that will have a popover
+		The regex has two capture groups, respectively the username and the path of the linked blog page.
+		https://xyz.tumblr.com links alway have a popover, while https://xyz.tumblr.com/foo/bar links do not have
+		a popover unless they're in a reblog header, in which case they have the post_info_link class.*/
+		var m_test = /https?:\/\/(?!www)([a-z0-9-]+)\.tumblr\.com([a-z0-9/-]*)/.exec(m_url);
```

Comment:
I haven't actually seen a blog like this in the wild, but I created a test blog whose URL begins with `www` (`http://wwwtheweb.tumblr.com`) and the regexp doesn't correctly pick up this case.